app/src/main/java/com/github/houseorganizer/houseorganizer/activity/MainScreenActivity.java

```diff
@@ -168,65 +172,14 @@ private void rotateView(View v) {
         calendarEvents.setLayoutManager(new GridLayoutManager(this, calendarColumns));
     }
 
-    private void addEvent(View v) {
-        LayoutInflater inflater = LayoutInflater.from(MainScreenActivity.this);
-        final View dialogView = inflater.inflate(R.layout.event_creation, null);
-        new AlertDialog.Builder(MainScreenActivity.this)
-                .setTitle(R.string.event_creation_title)
-                .setView(dialogView)
-                .setPositiveButton(R.string.add, (dialog, id) -> pushEventAndDismiss(dialog, dialogView, v))
-                .setNegativeButton(R.string.cancel, (dialog, id) -> dialog.dismiss())
-                .show();
+    private void refreshCalendar(View v) {
+        calendar.refreshCalendar(findViewById(R.id.refresh_calendar), db, currentHouse, calendarAdapter, Arrays.asList("MainScreenActivity",
+                "refreshCalendar:failureToRefresh"));
     }
```

Comment:
Since the method is moved into Calendar, you can delete this one and replace the calls in the class by calls to calendar.refreshCalendar, the same applies for the other methods that were moved

Reply:
I think changing line 84 to a lambda makes more sense than keeping the method here personally